### ISSUE TITLE
Improvement: after everything is done add ### __WORK IN PROGRESS__ to readme

### ISSUE THREAD

**Issue by Garfonso** [AlCalzone/release-script#6]
After everything is done as last action, add ### __WORK IN PROGRESS__ to Readme again.

Please. I have to look it up every single time... 😃

**Comment by AlCalzone** [comment]
I have a comment in my Readme where I copy and paste the placeholder from. Never thought about adding that automatically.

Good idea!

**Comment by AlCalzone** [comment]
Until I figure out how to best do this, you can work around this by adding a placeholder under the changelog headline. This is how I do it (or with one less `#` if you're using `CHANGELOG.md):
```md
## Changelog
<!-- 
	Placeholder for next versions:
	### __WORK IN PROGRESS__
-->

### 0.11.7 (2020-04-25)
```

**Comment by Apollon77** [comment]
I would add it into readme and do a separate commit afterwards (alternative would be to change the file but leave uncommited)

**Comment by AlCalzone** [comment]
Included in `v3.4.0`
